src/track/track.js

```diff
@@ -155,19 +156,18 @@ export default class Track extends Component {
   }
 
   handleKeyUp = ((nextKeys, prevKeys) => ({ key }) => {
-    const isNext = nextKeys.includes(key)
-    const isPrev = prevKeys.includes(key)
+    const isNext = includes(key, nextKeys)
+    const isPrev = includes(key, prevKeys)
     this.setState({ isAnimating: true })
     if (isNext) this.next().catch(noop)
     if (isPrev) this.prev().catch(noop)
     return false
   })(['ArrowRight'], ['ArrowLeft']);
 
-  // We don't want to update if only state changed.
-  // Sate is not important to the rendering of this component
-  shouldComponentUpdate (nextProps) {
-    const propValues = values(this.props)
-    const nextPropValues = values(nextProps)
+  // isAnimating state is the only important state value to the rendering of this component
```

Comment:
Does this mean some things should be moved out of `this.state` and just set on `this`?